src/rust/engine/process_execution/src/remote.rs

```diff
@@ -1219,6 +1219,7 @@ pub mod tests {
         .collect(),
       timeout: Duration::from_millis(1000),
       description: "some description".to_owned(),
+      local_only_scratch_files: hashing::EMPTY_DIGEST,
```

Comment:
Please add a test which has a non-empty digest here, and shows that it's ignored in all operations relating to the remote server

Reply:
Just realized that we only have mock server implementation, so this would depend on how scoot is working?

Reply:
Added in https://github.com/pantsbuild/pants/pull/8282/commits/a12f19af0002beda8ef8353652df839fc73a4c6a